website/app/layout.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC, ReactNode } from 'react';
+import { IBM_Plex_Sans } from 'next/font/google';
+import { Toaster } from 'react-hot-toast';
+import { Search } from '@theguild/components';
+import { getDefaultMetadata, GuildLayout } from '@theguild/components/server';
+import '@theguild/components/style.css';
+import './globals.css';
+
+const description = 'Modern, Open-source API Tooling and Ecosystem that scales';
+const websiteName = 'The Guild';
+
+export const metadata = getDefaultMetadata({
+  description,
+  websiteName,
+  productName: 'GUILD',
+  alternates: {
+    types: {
+      'application/rss+xml': [
+        {
+          title: 'RSS Feed for the-guild.dev',
+          url: '/feed.xml',
+        },
+      ],
+    },
+  },
+});
+
+const ibmPlexSans = IBM_Plex_Sans({
+  weight: ['400', '500', '600', '700'],
+  subsets: ['latin'],
+});
+
+const RootLayout: FC<{
+  children: ReactNode;
+}> = async ({ children }) => {
+  const jsonLd = {
+    '@context': 'https://schema.org',
+    '@type': 'Organization',
+    url: 'https://the-guild.dev',
+    logo: 'https://the-guild.dev/static/logo.svg',
+    name: 'The Guild',
+  };
```

Comment:
![CleanShot 2024-11-04 at 12 50 41@2x](https://github.com/user-attachments/assets/dcbd0bb1-c4c2-4459-a477-2c9c8643df73)


```suggestion
  const jsonLd = {
    '@context': 'https://schema.org',
    '@type': 'Organization',
    url: 'https://the-guild.dev',
    logo: 'https://the-guild.dev/static/logo.svg',
    name: 'The Guild',
    sameAs: ["https://x.com/TheGuildDev", "https://www.linkedin.com/company/the-guild-software", "https://github.com/the-guild-org"],
  };
```